From 1fe8ff3a40d2a9ffcd2be8c90aaf6644a119b8af Mon Sep 17 00:00:00 2001
From: placiano <placiano80@gmail.com>
Date: Sat, 25 Jul 2015 00:26:32 +0200
Subject: [PATCH] SM-N910F-CM: Patch default.prop

---
 SM-N910F_CM/default.prop | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SM-N910F_CM/default.prop b/SM-N910F_CM/default.prop
index 1324a5a..1f42a42 100644
--- a/SM-N910F_CM/default.prop
+++ b/SM-N910F_CM/default.prop
@@ -1,8 +1,8 @@
 #
 # ADDITIONAL_DEFAULT_PROPERTIES
 #
-ro.adb.secure=1
-ro.secure=1
+ro.adb.secure=0
+ro.secure=0
 ro.allow.mock.location=0
 ro.debuggable=1
 ro.zygote=zygote32
--
libgit2 0.22.2

